src/context/theme-context/ThemeContext.tsx

```diff
@@ -0,0 +1,47 @@
+import { createContext, useContext, useEffect, useState } from "react";
```

Comment:
Question: I can see you have theme context but the switch of the theme is not happening bug or not wired yet?